static/js/components/Logo.jsx

```diff
@@ -1,13 +1,34 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import styles from './Logo.css';
+import styled, { keyframes } from 'styled-components';
+
+
+const rotate = keyframes`
+  0% {
+    transform: rotate(60deg);
+  }
+  50% {
+    transform: rotate(-10deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+`;
+
+const StyledLogo = styled.img`
+  vertical-align: middle;
+  height: 50px;
+  margin-top: -0.5em;
+  animation-name: ${props => props.rotate ? rotate : null};
+  animation-duration: ${props => props.rotate ? '4s' : null};
+`;
 
 const Logo = () => {
-  const rotateLogo = useSelector((state) => state.logo.rotateLogo);
+  const rotateLogo = useSelector(state => state.logo.rotateLogo);
```

Comment:
the parameter parens is a really small thing (either way is 100% fine), this is just a personal preference as this feels easier to read.